src/app/services/config/config.service.ts

```
@@ -29,24 +29,24 @@
-const adminIP = 'http://10.208.122.38:8080/';
+const commonIP = 'https://amritwprdev.piramalswasthya.org/';
+const IP1097 = 'https://amritwprdev.piramalswasthya.org/';
+const telephonyServerIP = 'http://192.168.45.55/';
+const adminIP = 'https://amritwprdev.piramalswasthya.org/';
 
 @Injectable()
 export class ConfigService {
 
     // private _commonBaseURL: String = `${commonIP}commonapi-v1.0/`;
-    private _commonBaseURL: String = `http://10.208.122.38:8080/commonapi-v1.2/`;
+    private _commonBaseURL: String = `https://amritwprdev.piramalswasthya.org/commonapi-v3.0.0/`;
```

Biome (1.9.4)

[error] Don't use 'String' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'string' instead


(lint/complexity/noBannedTypes)